app/src/main/java/com/github/dedis/student20_pop/model/Person.java

```diff
@@ -0,0 +1,91 @@
+package com.github.dedis.student20_pop.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Class modeling a Person
+ */
+public class Person {
```

Comment:
Should this be final ? At least for now ?

Reply:
It is now